server/vulnerabilities/vuln_centos/centos.go

```diff
@@ -0,0 +1,512 @@
+// Package vuln_centos contains a ParseCentOSRepository method to parse the CentOS repository
```

Comment:
Nit: Instead of `vuln_centos` how about we call this package `linux` and have `centos.go` and `ubuntu.go` etc inside?

Reply:
+1, I'll re-consider when we start doing the post processing work for ubuntu.